### DISCUSSION TITLE
Custom sigma rules work for linux but not for windows

### DISCUSSION THREAD

**Discussion by Sarra2611** [Security-Onion-Solutions/securityonion#13444] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

64.44 GB

### Storage for /nsm

124.52

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, I've been trying to create my own sigma rules via the Detection interface and then enabling them. I created rules for both linux and windows. When I generate the traffic that should trigger the alerts, I find only the alerts related to the linux machines but none of the windows ones. The problem is that the rules are correct and when I convert them and test them in Kibana, I see that there is traffic and it is detected via the rules, but no alerts whatsoever are in the alerts interface. I tried on different windows machine to see if the problem resides in the host machine itself, but this still doesn't work. The traffic reaches the SOC and is found in kibana and the rule detects the events I want but the alerts aren't generated. I hope you can help me !

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment]
If possible, please post one of these custom rules here so that we can better assist you.
